docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+maturin
+sphinx
+sphinx-rtd-theme
+sphinxcontrib-napoleon
```

Comment:
This could be moved to pyproject as well

Reply:
yes it becames unnecessary if we use poetry to setup the enviroment. please check new commits.